test/test-server/org_handlers.go

```diff
@@ -64,6 +64,47 @@ func handleOrgEnvironments(t *testing.T) http.HandlerFunc {
 	}
 }
 
+// Handler for: "/org/v2/organizations/{id}"
+func handleOrgOrganization(t *testing.T) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		orgId := vars["id"]
```

Comment:
Probably a good idea to match the arg names, just like how we try to match flag names.
```suggestion
		id := vars["id"]
```